backend/ibutsu_server/controllers/widget_controller.py

```diff
@@ -79,6 +85,10 @@ def get_widget(id_):
     """
     if id_ not in WIDGET_TYPES.keys():
         return "Widget not found", 404
-    params = _pre_process_params(dict(connexion.request.args))
-    params = _typecast_params(id_, params)
+    else:
```

Comment:
I'm not sure why you added the `else` here? The `if` section will return, so the `else` is unnecessary, hence why it was left out originally.